console/services/team_services.py

```diff
@@ -386,9 +386,23 @@ def get_teams_region_by_user_id(self, enterprise_id, user, name=None):
         tenants = enterprise_repo.get_enterprise_user_teams(enterprise_id, user.user_id, name)
         if tenants:
             for tenant in tenants:
-                teams_list.append(self.__team_with_region_info(tenant, user))
+                team = self.__team_with_region_info(tenant, user)
+                team["join"] = True
```

Comment:
This field is redundant. If the role is empty, it is not join.

Reply:
In addition, the interface requires less loading of region information.

Reply:
Although it can be judged by roles, specifying a field will be more semantically clear

Reply:
The front end doesn't need to judge.